<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookItem</class>
 <widget class="QDialog" name="BookItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>730</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>440</y>
     <width>581</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="backButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton {
background-color:#6f1f27;
color: #a19798;
font: 16pt &quot;Arial&quot;;
border:2px solid #FFFFFF;
}

QPushButton:hover {
    background-color: #520a11;
}</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="rateButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton {
background-color:#6f1f27;
color: #a19798;
font: 16pt &quot;Arial&quot;;
border:2px solid #FFFFFF;
}

QPushButton:hover {
    background-color: #520a11;
}</string>
      </property>
      <property name="text">
       <string>Rate</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="downloadButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton {
background-color:#6f1f27;
color: #a19798;
font: 16pt &quot;Arial&quot;;
border:2px solid #FFFFFF;
}

QPushButton:hover {
    background-color: #520a11;
}</string>
      </property>
      <property name="text">
       <string>Download</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>701</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="isbn">
        <property name="styleSheet">
         <string notr="true">background-color:#565354; 
color: #a19798; 
font: 12pt Arial; 
border:2px solid #FFFFFF;</string>
        </property>
        <property name="text">
         <string>ISBN</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="isbnText">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#565354; 
color: #a19798; 
font: 14pt Arial; 
border:2px solid #FFFFFF;</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="title">
        <property name="styleSheet">
         <string notr="true">background-color:#565354; 
color: #a19798; 
font: 14pt Arial; 
border:2px solid #FFFFFF;</string>
        </property>
        <property name="text">
         <string>Title</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="titleText">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#565354; 
color: #a19798; 
font: 14pt Arial; 
border:2px solid #FFFFFF;</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="author">
        <property name="styleSheet">
         <string notr="true">background-color:#565354; 
color: #a19798; 
font: 14pt Arial; 
border:2px solid #FFFFFF;</string>
        </property>
        <property name="text">
         <string>Author</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="authorText">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#565354; 
color: #a19798; 
font: 14pt Arial; 
border:2px solid #FFFFFF;</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="yearOfPublication">
        <property name="styleSheet">
         <string notr="true">background-color:#565354; 
color: #a19798; 
font: 14pt Arial; 
border:2px solid #FFFFFF;</string>
        </property>
        <property name="text">
         <string>Year of Publication</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="yearOfPublicationText">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#565354; 
color: #a19798; 
font: 14pt Arial; 
border:2px solid #FFFFFF;</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="Rating">
        <property name="styleSheet">
         <string notr="true">background-color:#565354; 
color: #a19798; 
font: 14pt Arial; 
border:2px solid #FFFFFF;</string>
        </property>
        <property name="text">
         <string>Rating</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ratingText">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#565354; 
color: #a19798; 
font: 14pt Arial; 
border:2px solid #FFFFFF;</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="Genre">
        <property name="styleSheet">
         <string notr="true">background-color:#565354; 
color: #a19798; 
font: 14pt Arial; 
border:2px solid #FFFFFF;</string>
        </property>
        <property name="text">
         <string>Genre</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="genreText">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#565354; 
color: #a19798; 
font: 14pt Arial; 
border:2px solid #FFFFFF;</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="subgenre_2">
        <property name="styleSheet">
         <string notr="true">background-color:#565354; 
color: #a19798; 
font: 14pt Arial; 
border:2px solid #FFFFFF;</string>
        </property>
        <property name="text">
         <string>Subgenre</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="subgenreText">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#565354; 
color: #a19798; 
font: 14pt Arial; 
border:2px solid #FFFFFF;</string>
        </property>
        <property name="text">
         <string>Subgenre</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <property name="spacing">
       <number>7</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="QLabel" name="description">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#565354; 
color: #a19798; 
font: 14pt Arial; 
border:2px solid #FFFFFF;</string>
        </property>
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="descriptionText">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#565354; 
color: #a19798; 
font: 14pt Arial; 
border:2px solid #FFFFFF;</string>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
